code/modules/surgery/organs/organ_internal.dm

```diff
@@ -257,8 +257,20 @@
 	attack_verb = list("licked", "slobbered", "slapped", "frenched", "tongued")
 
 /obj/item/organ/internal/tongue/proc/TongueSpeech(var/message)
+	spawn(0)
+		speakDevilName(message)
 	return message
 
+/obj/item/organ/internal/tongue/proc/speakDevilName(var/message)
```

Comment:
How about implementing speech activated spells, then giving every living mob the SUMMON THE DEMON spell secretly. Still means you have speech scanning, but in a more general form.